apache/trafficcontrol
# Update Go to version 1.14

## What does this PR (Pull Request) do?


This PR:
* Updates Go to 1.14 for all component
* Enables TLS version 1.1 in Traffic Vault



## Which Traffic Control components are affected by this PR?


- CDN in a Box
- Documentation
- Grove
- Traffic Control Client (Go)
- Traffic Monitor
- Traffic Ops
- Traffic Ops ORT
- Traffic Stats
- Traffic Vault

## What is the best way to verify this PR?

- Run unit tests
- Run Traffic Ops API tests
- Make sure Traffic Vault still works
- Check for Go 1.14 deprecations in dependencies:
    ```shell script
    staticcheck -checks SA1019 ./... | grep -vF ': tc.'
    ```
- Make sure docs build with no warnings
  
The MaxTLSVersion config setting for `riak.conf` has a test. Besides that, this PR relies on the tests that are already in place to check for breaking changes when updating the Go version.

## The following criteria are ALL met by this PR
